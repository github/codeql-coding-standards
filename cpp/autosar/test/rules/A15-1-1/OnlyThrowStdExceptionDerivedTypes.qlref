rules/A15-1-1/OnlyThrowStdExceptionDerivedTypes.ql