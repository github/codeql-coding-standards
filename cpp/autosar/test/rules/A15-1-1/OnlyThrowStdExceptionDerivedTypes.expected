| test.cpp:9:5:9:12 | throw ... | Exception thrown of type $@ which is not derived from std::exception. | file://:0:0:0:0 | char * | char * |
| test.cpp:11:5:11:18 | throw ... | Exception thrown of type $@ which is not derived from std::exception. | test.cpp:3:7:3:12 | ClassA | ClassA |
